### DISCUSSION TITLE
Documents Not Detected

### DISCUSSION THREAD

**Discussion by yzdnaufan** [open-webui/open-webui#4641] [Q&A]
Hi, when I tried to use with OpenAI model to upload document, the model doesn't recognize the document uploaded in the workspace. Any help please? Maybe I got something missing because I could not find similar issue across the discussions.

<img width="1274" alt="image" src="https://github.com/user-attachments/assets/59a456a4-4455-4616-8f0b-99760d227706">

Here is also my documents config

<img width="1266" alt="image" src="https://github.com/user-attachments/assets/857afbfb-1136-46fc-b85c-89daab971215">

**Comment by yzdnaufan** [comment]
I've tried solution provided in #3868 but it doesn't seems work for case using GPT-4o

<img width="1275" alt="image" src="https://github.com/user-attachments/assets/6012c161-98e1-48ab-9fc0-2b69c64e167f">

**Comment by HANIHALILI** [comment]
I’ve encountered the same issue.
In my case, the solution was to click the “try again to answer” button (the one with the arrows) to make the model recognize the file.


I’ve encountered the same issue.
In my case, the solution was to click the “try again to answer” button (the one with the arrows) to make the model  recognize the file and respond properly.

It seems the problem might be caused by high server load — even though the file upload looks complete, the server is still processing it in the background.
Since the RAG can’t query the file until analysis is done, the best workaround for now is simply to wait a bit before trying again.
